.github/workflows/ci.yml

```
@@ -40,11 +40,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.11", "3.12"]
+        python-version: ["3.9", "3.11", "3.12", "3.13"]
```

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
```

missing Python 3.10 too.